language_lilypond.lua

```diff
@@ -0,0 +1,19 @@
+-- mod-version:3
+local syntax = require "core.syntax"
+
+syntax.add {
+  name = "LilyPond",
+  files = { "%.ly$" },
```

Comment:
Seems like another possible extension is `.ily`, so you could do:
```suggestion
  files = { "%.i?ly$" },
```